### PR TITLE
feat(localization): Create stream_chat_localizations_no.dart 

### PR DESCRIPTION
**Pull request by monhelle**
feat: Added the Norwegian translation for the Stream Chat

# Submit a pull request

## CLA

- [x ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)
I copied and pasted the english version of the language translations and translated directly between the apostrophes.

### CODE REVIEW

**Comment by imtoori** [comment]
@monhelle can you fix the analysis errors?

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart [hunk 1]
```diff
@@ -0,0 +1,392 @@
+part of 'stream_chat_localizations.dart';
+
+/// The translations for English (`en`).
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart:3]:
```suggestion
/// The translations for Norwegian (`no`).
```

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+class StreamChatLocalizationsEn extends GlobalStreamChatLocalizations {
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart:4]:
```suggestion
class StreamChatLocalizationsNo extends GlobalStreamChatLocalizations {
```

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+  /// Create an instance of the translation bundle for English.
+  const StreamChatLocalizationsEn({super.localeName = 'no'});
```
Comment (xsahil03x) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_no.dart:6]:
```suggestion
  const StreamChatLocalizationsNo({super.localeName = 'no'});
```

**Comment by xsahil03x** [comment]
Hey @monhelle, you also need to update the [stream_chat_localizations.dart](https://github.com/GetStream/stream-chat-flutter/blob/develop/packages/stream_chat_localizations/lib/src/stream_chat_localizations.dart) in order for it to work.

**Comment by monhelle** [comment]
> [.at_flutter_core/lib/src/lazy_load_scroll_view.dart](https://codecov.io/gh/GetStream/stream-chat-flutter/pull/1218/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=GetStream#diff-cGFja2FnZXMvc3RyZWFtX2NoYXRfZmx1dHRlcl9jb3JlL2xpYi9zcmMvbGF6eV9sb2FkX3Njcm9sbF92aWV3LmRhcnQ=)	`100.00% <0.00%> (ø)`	
> [.../channel_scroll_view/stream_channel_list_tile.dart](https://codecov.io/gh/GetStream/stream-chat-flutter/pull/1218/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=GetStream#diff-cGFja2FnZXMvc3RyZWFtX2NoYXRfZmx1dHRlci9saWIvc3JjL3Y0L3Njcm9sbF92aWV3L2NoYW5uZWxfc2Nyb2xsX3ZpZXcvc3RyZWFtX2NoYW5uZWxfbGlzdF90aWxlLmRhcnQ=)	`0.00% <0.00%> (ø)`

I am not entirely sure what I am going to do with these two files 🐙 

@xsahil03x  thank you for mentioning, I have now updated this file :)

**Pull request merged**